examples/north_sea/model_config.py

```diff
@@ -0,0 +1,192 @@
+from thetis import *
+import thetis.coordsys as coordsys
+import thetis.forcing as forcing
+import csv
+import netCDF4
+import numpy
+import os
+import pyproj
+import scipy.interpolate as si
+
+
+# Setup zones
+sim_tz = timezone.pytz.utc
+UTM_ZONE30 = pyproj.Proj(proj="utm", zone=30, datum="WGS84", units="m", errcheck=True)
+
+# Load tide gauge metadata
+with open("stations_elev.csv", "r") as csvfile:
+    stations = {
+        d["name"]: {
+            "latlon": (float(d["latitude"]), float(d["longitude"])),
+            "region": d["region"],
+        }
+        for d in csv.DictReader(csvfile, delimiter=",", skipinitialspace=True)
+    }
```

Comment:
This is executed on import. Better to wrap it in a function? AFAIK it's only used in `construct_solver`